### DISCUSSION TITLE
SteamConfiguration and IP addresses

### DISCUSSION THREAD

**Discussion by MisterGibs** [SteamRE/SteamKit#1304] [General]
var configuration = SteamConfiguration.Create(b =>
{
	b.WithProtocolTypes(ProtocolTypes.All);
});
client = new SteamClient(configuration);

I have several network interfaces (i.e. several IP addresses), how can I explicitly indicate from which IP address I want to connect to Steam?

**Comment by yaakov-h** [comment]
You can't at the moment.

This would be doable as a new feature, but it wouldn't be able to be nicely done across both TCP/UDP and WebSockets - they would need their own configuration.

**Reply by MisterGibs** [reply]
Can you help me do this?

**Comment by Fess23** [comment]
up
